src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -84,44 +84,43 @@
-                                <a asp-show="@vacancy.IsSubmittable" asp-route="@RouteNames.DisplayVacancy_Get" asp-route-vacancyId="@vacancy.Id" class="govuk-link">@GetLinkText(vacancy)</a>
-                            </td>
-                        </tr>
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell" data-label="Vacancy title">
+                            <div>@vacancy.Title</div>
+                        </td>
+                        <td class="govuk-table__cell" data-label="Vacancy ref">
+                            <span asp-show="@vacancy.HasVacancyReference"><span>VAC</span>@vacancy.VacancyReference</span>
+                            <span asp-show="@vacancy.HasNoVacancyReference">Not available</span>
+                        </td>
+                        <td asp-show="@vacancy.CanShowVacancyApplicationsCount" class="govuk-table__cell dashboard-applications" data-label="Applications">
+                            <span asp-show="@vacancy.HasNoApplications">0</span>
+                            <span asp-show="@vacancy.HasApplications">@vacancy.NoOfApplications</span>
+                            <span asp-show="@vacancy.HasNewApplications" class="govuk-!-font-weight-bold"> (@vacancy.NoOfNewApplications new)</span>
+                        </td>
+                        <td asp-hide="@vacancy.CanShowVacancyApplicationsCount" class="govuk-table__cell dashboard-applications" data-label="Applications">-</td>
+                        <td class="govuk-table__cell dashboard-closingdate" data-label="Closing date"><span>@vacancy.ClosingDate?.AsGdsDate()</span></td>
+                        <td class="govuk-table__cell dashboard-status" data-label="Status"><span class="tag tag-@vacancy.Status.ToString().ToLower()">@vacancy.Status.GetDisplayName()</span></td>
+                        <td class="govuk-table__cell dashboard-action" data-label="Action">
+                            <a asp-show="@vacancy.IsNotSubmittable" asp-route="@RouteNames.VacancyManage_Get" asp-route-vacancyId="@vacancy.Id" class="govuk-link">Manage</a>
+                            <a asp-show="@vacancy.IsSubmittable" asp-route="@RouteNames.DisplayVacancy_Get" asp-route-vacancyId="@vacancy.Id" class="govuk-link">@GetLinkText(vacancy)</a>
+                        </td>
+                    </tr>
                     }
-                    </tbody>
-                </table>
-                <partial name="@PartialNames.Pager" model="@Model.Pager"/>
-            </div>
+                </tbody>
+            </table>
+            <partial name="@PartialNames.Pager" model="@Model.Pager" />
         </div>
     </div>
 </div>
+</div>
 
 @section FooterJS
 {
-<script nws-csp-add-nonce="true">
-    var filterId = "#@Html.IdFor(m => m.Filter)";
+<script nws-csp-add-nonce="true">var filterId = "#@Html.IdFor(m => m.Filter)";
```

Comment:
As discussed. Can you undo these formatting changes.  I think the only change we should have is the change to the back link.